include/inexor/vulkan-renderer/wrapper/cpu_texture.hpp

```diff
@@ -39,31 +39,31 @@ class CpuTexture {
     CpuTexture &operator=(const CpuTexture &) = delete;
     CpuTexture &operator=(CpuTexture &&) = default;
 
-    [[nodiscard]] std::string name() const {
+    [[nodiscard]] std::string name() const noexcept {
```

Comment:
Shouldn't be `noexcept` since it can throw.

Reply:
Can it?
Also `m_name` can be const.

Reply:
Yes, constructing a string may throw `std::length_error` or `std::bad_alloc`.

Reply:
I will drop the commit which introduces `noexcept` in get methods as it gives us no benefits.